src/corehost/cli/hostpolicy.cpp

```diff
@@ -112,7 +113,7 @@ int run(const arguments_t& args)
     // Bind CoreCLR
     if (!coreclr::bind(clr_path))
     {
-        trace::error(_X("Failed to bind to coreclr"));
+        trace::error(_X("Failed to bind to coreclr at [%s]"), clr_path.c_str());
```

Comment:
We use `CoreCLR` and `coreclr` a little inconsistently in messages. It looks like `CoreCLR` is more common so let's align to that.